src/angular/tooltip/tooltip.scss

```diff
@@ -45,43 +40,40 @@ $tooltipShadowArrowSizeLean: 13.333;
   }
   :where(html.sbb-lean) &::after {
     bottom: pxToRem(8.25);
-    height: pxToRem($tooltipContentArrowSizeLean);
-    width: pxToRem($tooltipContentArrowSizeLean);
-    border-width: pxToRem(1);
-    border-right-color: var(--sbb-color-iron);
-    border-bottom-color: var(--sbb-color-iron);
   }
 }
 
 .sbb-tooltip {
   max-width: pxToEm(460);
-  padding: pxToEm(23);
+  padding: var(--sbb-tooltip-padding);
   overflow: hidden;
   color: var(--sbb-color-grey);
   text-overflow: ellipsis;
-  background-color: var(--sbb-color-white);
-  border: pxToEm(2) solid var(--sbb-color-grey);
-  border-radius: pxToEm(2);
+  background-color: var(--sbb-color-background);
+  border: var(--sbb-border-width) solid var(--sbb-tooltip-border-color);
+  border-radius: var(--sbb-border-radius);
 
   @include cdk.high-contrast(active, off) {
     outline: solid 1px;
   }
 
   .sbb-tooltip-panel-below & {
-    margin-top: pxToEm(7.2);
+    margin-top: var(--sbb-tooltip-arrow-offset);
   }
 
   :where(html.sbb-lean) & {
-    color: var(--sbb-color-black);
-    padding: pxToRem(8);
-    border: pxToRem(1) solid var(--sbb-color-iron);
+    color: var(--sbb-color-text);
   }
 }
 
 .sbb-tooltip-shadow {
-  height: pxToEm(8);
-  background-color: rgba(0, 0, 0, 0.15);
-  border-radius: 0 0 pxToEm(2) pxToEm(2);
+  height: var(--sbb-box-shadow-width);
+  background-color: var(--sbb-box-shadow-color);
+  border-radius: 0 0 var(--sbb-border-radius) var(--sbb-border-radius);
+
+  :where(html.sbb-lean) & {
+    border-radius: 0 0 pxToRem(1) pxToRem(1);
```

Comment:
I think 2px would be fine here too.

General problem, the shadow should be taller:
height: calc(var(--sbb-box-shadow-width) + var(--sbb-border-radius));
margin-top: calc(-1 * var(--sbb-border-radius);

and as a following problem, stacking of arrow, content and shadow has to be resolved, probably by z-index and position absolute.

Reply:
<img width="150" alt="Bildschirmfoto 2021-12-01 um 09 15 36" src="https://user-images.githubusercontent.com/29278232/144196871-206e823c-f5d8-492e-9aac-8967ee9edd43.png">